README.adoc

```diff
@@ -18,7 +18,7 @@ You can access the stable release from maven central using the coordinates
 <dependency>
```

Comment:
On the topic of `README`, since the project will have 2 branches, I think it would help to explain them in the README.
For example in app-broker https://github.com/spring-cloud/spring-cloud-app-broker there's a sections explaining what versions are supported and which version of spring boot are related.

Reply:
ok!